.pre-commit-config.yaml

```diff
@@ -4,8 +4,8 @@ fail_fast: false
 default_language_version:
     python: python3
 default_stages:
--   commit
--   push
+-   pre-commit
```

Comment:
Please explain the change here and be more descriptive on your commit message, your commit message should talks about this part as well.

Reply:
I would even suggest to revert this and make the subject of a separate issue and PR. I have not thought deeply about the pre-commit setup, but this is certainly something we can discuss (in combination with, for instance, whether documentation changes should trigger the test).

Reply:
Yes, see [https://github.com/biocypher/biocypher/pull/379#issuecomment-2407561946](url)